src/containers/WhiteContainer.tsx

```diff
@@ -5,7 +5,7 @@ import ButtonContainer from "components/Form/FormButton/ButtonContainer";
 
 interface IProps {
   step: number;
-  children: JSX.Element;
+  children: JSX.Element | JSX.Element[];
```

Comment:
That's not broad enough of a definition. The child could be a string.
 I would recommend using children: React.ReactNode instead now.

![image](https://user-images.githubusercontent.com/108327339/186436326-98d7bd64-3e54-499d-87d6-585006285dad.png)